macros/stitch/base/stitch_bing_keyword_performance.sql

```diff
@@ -9,50 +9,69 @@
 with source as (
 
     select * from {{var('keyword_performance_report_table')}}
-    
+
 ),
 
 renamed as (
 
     select
-    
+
         "__SDC_PRIMARY_KEY" as keyword_performance_report_id,
 
-        convert_timezone('UTC', timeperiod)::timestamp_ntz::date 
-            as campaign_date,
-            
+        {% if target.type == 'bigquery' %}
```

Comment:
We tend to use the "adapter macro" pattern to render adapter-specific logic in packages. This has the advantage of avoiding really long chains of `if/else`statements in dbt models.

This isn't so well documented at the moment, but you can find an example here: https://github.com/fishtown-analytics/dbt-utils/blob/master/macros/cross_db_utils/hash.sql

This macro can be used like:
```
select {{ dbt_utils.hash('some_field') }}
```

Behind the scenes, dbt will dispatch the macro call to either `bigquery__hash` or `default__hash`, depending on the target database.

In this case, I think we'd either want to make one such macro called `convert_to_utc`.